tests/test_record_resource.py

```diff
@@ -59,7 +59,7 @@ def test_response_contents(self, response, endpoint, record_schema_v2):
         register_data = requests.get(urljoin(endpoint, 'register.json'))
         register_fields = register_data.json()['register-record']['fields']
 
-        assert set(record_json['blob'].keys()).issubset(register_fields), \
+        assert set(record_json.keys()).issubset(register_fields + ['_id']), \
```

Comment:
I think this is not quite correct because `_id` should always be there.

Maybe we can do something like this:

```python
assert '_id' in record_json

names = set(record_json.keys()) - {'_id'}
assert names.issubset(register_fields))
```